### ISSUE TITLE
Incorrect work of LogLoss function

### ISSUE THREAD

**Issue by zaleslaw** [JetBrains/KotlinDL#2]
Exception in thread "main" java.lang.IllegalStateException: Tensor is not a scalar
for   it.compile(optimizer = SGD(LEARNING_RATE), loss = LossFunctions.LogLoss) in LeNetMnist example
